python/python/tests/test_async_client.py

```diff
@@ -909,6 +909,44 @@ async def test_rpushx(self, redis_client: TRedisClient):
         with pytest.raises(RequestError):
             await redis_client.rpushx(key2, [])
 
+    @pytest.mark.parametrize("cluster_mode", [True, False])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_blpop(self, redis_client: TRedisClient):
+        key1 = f"{{test}}-f{get_random_string(10)}"
+        key2 = f"{{test}}-f{get_random_string(10)}"
```

Comment:
nit: this simplifies debugging
```suggestion
        key1 = f"{{test}}-1-f{get_random_string(10)}"
        key2 = f"{{test}}-2-f{get_random_string(10)}"
```